<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>750</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Financial manager</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>üëâ</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb1">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ËÆ∞ÂΩïÊìç‰Ωú</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Êñá‰ª∂Êìç‰Ωú</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Â∏ÆÂä©</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb11">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ê∑ªÂä†ËÆ∞ÂΩï</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb12">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ÁºñËæëËÆ∞ÂΩï</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Âà†Èô§ËÆ∞ÂΩï</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb14">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Êü•ÁúãËÆ∞ÂΩï</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb15">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>ÁîüÊàêÊä•Âëä</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb21">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‰ªéÊñá‰ª∂ËØªÂÖ•</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb22">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>80</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>‰øùÂ≠òÂà∞Êñá‰ª∂</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb31">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Â∏ÆÂä©</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
